templates/test/ci/prow-azl3/patches/azuremachinetemplate-azl3-image.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
+kind: AzureMachineTemplate
+metadata:
+  name: ${CLUSTER_NAME}-control-plane
+spec:
+  template:
+    spec:
+      image:
+        computeGallery:
+          gallery: ClusterAPI-f72ceb4f-5159-4c26-a0fe-2ea738f0d019
+          name: capi-azurelinux-3
+          version: ${AZL3_VERSION:="1.33.2"}
```

Comment:
This will drift over time (it already has: v1.33.4 is out). Maybe we could use or improve the "get latest version" utils so this could float, like `v1.33-stable`.

Reply:
That's a good idea! I forgot I hard-coded this value as a part of testing. Let me get that fixed up.

Reply:
It is set to KUBERNETES_VERSION in azure_test.go anyways so I'll remove the default! The only other place this variable is used is in the presubmit job in test-infra which sets AZL3_VERSION in the spec